index.html

```diff
@@ -25,9 +25,8 @@
   <main>
     <header class="nav-links">
       <div class="logo">
-        <img class="img-logo" src="images/logo.svg" alt="logo">
+        <a href="#"><img class="img-logo" src="images/logo.svg" alt="logo"></a>
```

Comment:
- [ ] Great job for wrapping it with a tag but as asked by the previous reviewer you should link it to the home page since you wrote in the href just # but you should link it with home page to  allows users to easily navigate to the homepage of a website by clicking on the logo ✨🤩🙏🏻 so please link it

Reply:
@Reem-lab please I know you're doing a good job, but when I click on the logo that's gonna lead me to the home page 🤦‍♂️.

Reply:
- [ ] Hello @aradradev whenI try to press on it it doesn't take me to the home page